app/components/ch/components/Table.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div class="table-container">
+    <table class="table">
+      <thead class="table__header">
+        <th class="table__header--entry">Name</th>
```

Comment:
the class name `table__header--entry` is misleading if we stick to pure BEM. the double dash define a modifier. It should be used to overwrite `table__header` and shouldn't be used for its child.  
More generally I suggest to remove the class names for the tables, as they all have to be styled per default. Additionally, the users may create tables trough wysiwyg components and it may be complicated to inject classnames everywhere. WDYT?

Reply:
I also thought about just using the html table but at the end i wanted to implement a more general way. Your point with the wysiwyg editor is a good one. The css will often be used with a cms and its not that easy to add classes there. Lets change it to raw html elements

Reply:
Switched to the plain html elements